specs/helper.js

```diff
@@ -182,7 +182,7 @@ const dispatchMockEvent = eventCtor => (key, code) => (element, opts) =>
       {},
       {
         key: key,
-        keyCode: code,
+        code: code,
```

Comment:
I'm not really familiar with the testing setup in this project—let me know if this test util should be amended differently 🙂

Reply:
We need to change the arg in this part `(key, code)` where `code` needs to be an object like `{ keyCode: ... }` or `{ code: ... }`, so we can emulate both behaviors.

Reply:
Following up in https://github.com/reactjs/react-modal/pull/953/files#r896140750